### ISSUE TITLE
The problem of the AUC

### ISSUE THREAD

**Issue by Sun-Fan** [ekosman/AnomalyDetectionCVPR2018-Pytorch#6]
Thanks for the codes.It really helps me a lot. But i only can get the 0.73 AUC using your codes and your result is 0.74?  It is still a little lower than the paper result which is 0.75 AUC. Can you give me some advices？ Thanks again

**Comment by henrryzh1** [comment]
> Thanks for the codes.It really helps me a lot. But i only can get the 0.73 AUC using your codes and your result is 0.74? It is still a little lower than the paper result which is 0.75 AUC. Can you give me some advices？ Thanks again
Did you use the shared features or your extracted features? I use myself features,it can only get 70%, but it can get 74% when i use the shared features(I change the optimised algorithms following the paper----Adagrad)

**Comment by Sun-Fan** [comment]
> > Thanks for the codes.It really helps me a lot. But i only can get the 0.73 AUC using your codes and your result is 0.74? It is still a little lower than the paper result which is 0.75 AUC. Can you give me some advices？ Thanks again
> > Did you use the shared features or your extracted features? I use myself features,it can only get 70%, but it can get 74% when i use the shared features(I change the optimised algorithms following the paper----Adagrad)

Thanks a lot, the features i use is the shared features. And  I will try the Adagrad.

**Comment by Sun-Fan** [comment]
> > > Thanks for the codes.It really helps me a lot. But i only can get the 0.73 AUC using your codes and your result is 0.74? It is still a little lower than the paper result which is 0.75 AUC. Can you give me some advices？ Thanks again
> > > Did you use the shared features or your extracted features? I use myself features,it can only get 70%, but it can get 74% when i use the shared features(I change the optimised algorithms following the paper----Adagrad)
> 
> Thanks a lot, the features i use is the shared features. And I will try the Adagrad.

Another question, do you use the initial learning rate 0.001 as the paper when using the Adagrad.

**Comment by henrryzh1** [comment]
> > > > Thanks for the codes.It really helps me a lot. But i only can get the 0.73 AUC using your codes and your result is 0.74? It is still a little lower than the paper result which is 0.75 AUC. Can you give me some advices？ Thanks again
> > > > Did you use the shared features or your extracted features? I use myself features,it can only get 70%, but it can get 74% when i use the shared features(I change the optimised algorithms following the paper----Adagrad)
> > 
> > 
> > Thanks a lot, the features i use is the shared features. And I will try the Adagrad.
> 
> Another question, do you use the initial learning rate 0.001 as the paper when using the Adagrad.

Yes,waiting for your result with your own features.

**Comment by Sun-Fan** [comment]
> > > > > Thanks for the codes.It really helps me a lot. But i only can get the 0.73 AUC using your codes and your result is 0.74? It is still a little lower than the paper result which is 0.75 AUC. Can you give me some advices？ Thanks again
> > > > > Did you use the shared features or your extracted features? I use myself features,it can only get 70%, but it can get 74% when i use the shared features(I change the optimised algorithms following the paper----Adagrad)
> > > 
> > > 
> > > Thanks a lot, the features i use is the shared features. And I will try the Adagrad.
> > 
> > 
> > Another question, do you use the initial learning rate 0.001 as the paper when using the Adagrad.
> 
> Yes,waiting for your result with your own features.

ok

**Comment by haoranD** [comment]
Wonderful work and interesting discussion!

Can I ask a question about extracting the features by C3D model?
When I ran the feature_extractor.py with C3D model, I found that all my memories(32GB) were occupied and it seems I need more memories.

Did you run into this kind of problems?
Can you please give me some advice?

Thank you.

**Comment by Sun-Fan** [comment]
> > > > > Thanks for the codes.It really helps me a lot. But i only can get the 0.73 AUC using your codes and your result is 0.74? It is still a little lower than the paper result which is 0.75 AUC. Can you give me some advices？ Thanks again
> > > > > Did you use the shared features or your extracted features? I use myself features,it can only get 70%, but it can get 74% when i use the shared features(I change the optimised algorithms following the paper----Adagrad)
> > > 
> > > 
> > > Thanks a lot, the features i use is the shared features. And I will try the Adagrad.
> > 
> > 
> > Another question, do you use the initial learning rate 0.001 as the paper when using the Adagrad.
> 
> Yes,waiting for your result with your own features.

I have tried the Adagrad with the 0.001 learning rate, but the result is still 0.73. Can we have more communication？

**Comment by henrryzh1** [comment]
> > > > > > Thanks for the codes.It really helps me a lot. But i only can get the 0.73 AUC using your codes and your result is 0.74? It is still a little lower than the paper result which is 0.75 AUC. Can you give me some advices？ Thanks again
> > > > > > Did you use the shared features or your extracted features? I use myself features,it can only get 70%, but it can get 74% when i use the shared features(I change the optimised algorithms following the paper----Adagrad)
> > > > 
> > > > 
> > > > Thanks a lot, the features i use is the shared features. And I will try the Adagrad.
> > > 
> > > 
> > > Another question, do you use the initial learning rate 0.001 as the paper when using the Adagrad.
> > 
> > 
> > Yes,waiting for your result with your own features.
> 
> I have tried the Adagrad with the 0.001 learning rate, but the result is still 0.73. Can we have more communication？

Yes, maybe you can mail me. 913513866@qq.com. Also you can add my qq.

**Comment by haoranD** [comment]
> > > > > > > Thanks for the codes.It really helps me a lot. But i only can get the 0.73 AUC using your codes and your result is 0.74? It is still a little lower than the paper result which is 0.75 AUC. Can you give me some advices？ Thanks again
> > > > > > > Did you use the shared features or your extracted features? I use myself features,it can only get 70%, but it can get 74% when i use the shared features(I change the optimised algorithms following the paper----Adagrad)
> > > > > 
> > > > > 
> > > > > Thanks a lot, the features i use is the shared features. And I will try the Adagrad.
> > > > 
> > > > 
> > > > Another question, do you use the initial learning rate 0.001 as the paper when using the Adagrad.
> > > 
> > > 
> > > Yes,waiting for your result with your own features.
> > 
> > 
> > I have tried the Adagrad with the 0.001 learning rate, but the result is still 0.73. Can we have more communication？
> 
> Yes, maybe you can mail me. [913513866@qq.com](mailto:913513866@qq.com). Also you can add my qq.

hi, I don't know the answer to your QQ friends adding, can you please add mine? 794761554

**Comment by Sun-Fan** [comment]
> > > > > > > Thanks for the codes.It really helps me a lot. But i only can get the 0.73 AUC using your codes and your result is 0.74? It is still a little lower than the paper result which is 0.75 AUC. Can you give me some advices？ Thanks again
> > > > > > > Did you use the shared features or your extracted features? I use myself features,it can only get 70%, but it can get 74% when i use the shared features(I change the optimised algorithms following the paper----Adagrad)
> > > > > 
> > > > > 
> > > > > Thanks a lot, the features i use is the shared features. And I will try the Adagrad.
> > > > 
> > > > 
> > > > Another question, do you use the initial learning rate 0.001 as the paper when using the Adagrad.
> > > 
> > > 
> > > Yes,waiting for your result with your own features.
> > 
> > 
> > I have tried the Adagrad with the 0.001 learning rate, but the result is still 0.73. Can we have more communication？
> 
> Yes, maybe you can mail me. [913513866@qq.com](mailto:913513866@qq.com). Also you can add my qq.

my qq is 1019651257，i can't answer your qq question.

**Comment by PHDPeter** [comment]
> Wonderful work and interesting discussion!
> 
> Can I ask a question about extracting the features by C3D model?
> When I ran the feature_extractor.py with C3D model, I found that all my memories(32GB) were occupied and it seems I need more memories.
> 
> Did you run into this kind of problems?
> Can you please give me some advice?
> 
> Thank you.

could be this issue, (https://github.com/ekosman/AnomalyDetectionCVPR2018-Pytorch/issues/7) I have put in a solve for low memory setups if it is the issue.
Hope this helps.
